media/css/firefox/features/article.scss

```diff
@@ -9,6 +9,31 @@ $image-path: '/media/protocol/img';
 @import '~@mozilla-protocol/core/protocol/css/components/callout';
 @import '~@mozilla-protocol/core/protocol/css/components/breadcrumb';
 
+.mzp-c-breadcrumb {
+    padding-top: $spacing-md;
+    padding-bottom: $spacing-md;
+
+    .mzp-c-breadcrumb-item a {
+        color: inherit;
```

Comment:
**Nit:** By changing the color of the link, it doesn't make it obvious that it's a link a user can interact with anymore, so it's best to remove this declaration to not make it become an accessibility issue

Reply:
Thanks @reemhamz but I can't take any credit for this 😇 it's just repurposing the code `@nathan-barrett` originally wrote for `stories/articles`, and I didn't want it to disappear in git history now that the content is gone;)

But you're right, the styling should be more generic, and also cover all the occurrences of the breadcrumbs (VPN articles is a good example where the bad alignment shows best) so I'm backing out of this local change, and starting over together with fixes to subnav, basically treating the subnav+breadcrumbs containers as close as possible to the masthead, not the content defaults they inherit.

Thanks for the hand-holding, I really appreciate it! Sure I've read about the bundles needing restarts but have forgotten about it since;) so better to get reminded a few times than scratching my head why can't get it loaded;)

I'm trying to improve things that I feel comfortable with, either from what I can infer from some historic patterns, or current epics — as not to bother anyone else too much, having pretty good idea what all you have to juggle… — so you can focus on what _needs_ to be done, and I'm toying with random cosmetic stuff that I can either reliably resolve, or bail out fast if any broader involvement is needed. So I'll try to do my best to only bring up important decisions, not bothering you with me tripping over things, that's for me to figure out… (let's call it a learning curve, _one pebble at a time_, right?)

I'll see if I can apply fixes without introducing any new bundles (as subnav is included everywhere important, in base; and historically breadcrumbs never needed any custom styles so I'd be awkward for everyone to get used to including a bundle for them everywhere now…) and once the layout is something folks would like to see happening, we can find the best place where to keep the patches.